app/assets/javascripts/oxalis/view/tracing_layout_view.js

```diff
@@ -44,6 +45,12 @@ class TracingLayoutView extends React.PureComponent<Props, State> {
     });
   };
 
+  componentDidCatch() {
+    Toast.error(
```

Comment:
We could `console.error` before the toast, so the error would be visible in the console as well (no need to break).

Reply:
The error is logged either way, so that shouldn't be necessary :)